pkg/cmd/plan.go

```diff
@@ -0,0 +1,212 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"text/template"
+
+	"github.com/ipfs/testground/pkg/api"
+	"github.com/ipfs/testground/pkg/config"
+
+	"github.com/BurntSushi/toml"
+	"github.com/go-git/go-git/v5"
+	gitcfg "github.com/go-git/go-git/v5/config"
+	"github.com/go-git/go-git/v5/plumbing"
+	"github.com/mattn/go-zglob"
+	"github.com/urfave/cli/v2"
+)
+
+var PlanCommand = cli.Command{
+	Name:  "plan",
+	Usage: "plan management",
+	Subcommands: cli.Commands{
+		&cli.Command{
+			Name:  "create",
+			Usage: "create plan `PLAN_NAME`",
+			Flags: []cli.Flag{
+				&cli.StringFlag{
+					Name:     "module",
+					Usage:    "Create module named `MODULE_NAME`",
+					Required: false,
+					Value:    "github.com/your/module/name",
+				},
+			},
+			Action: createCommand,
+		},
+		&cli.Command{
+			Name:   "import",
+			Usage:  "`GIT_REPO` [local_repo]",
+			Action: importCommand,
+		},
+		&cli.Command{
+			Name:   "branch",
+			Usage:  "`LOCAL_REPO` `BRANCH_NAME`",
+			Action: importCommand,
+		},
```

Comment:
We discussed omitting the branch command. There are too many edge cases, and it's likely that the user will want to manage branches directly. Also, it only applies to Git origins only (we want the `testground plan` command to be able to manage local symlinks).

Reply:
@coryschwartz ?

Reply:
I removed the branch command -- yeah, it didn't work with local imports. Additionally, I enabled git with a flag `--git` so that it would work even with local bare repos!